packages/esm-home-app/src/root.scss

```diff
@@ -1 +1,2 @@
-@import "carbon-components/scss/globals/scss/typography.scss";
+@use "@carbon/styles/scss/type";
+@use "@carbon/styles/scss/spacing";
```

Comment:
```suggestion
@use "@carbon/styles/scss/type";
@use "@carbon/styles/scss/spacing";
```

We don't need these imports because Carbon isn't used anywhere in this repository.

Reply:
@denniskigen do u recommend we leave it empty?

Reply:
Yes

Reply:
Done